backend/business-partner-agent/src/main/java/org/hyperledger/bpa/client/BPALivenessCheck.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2020-2021 - for information on the respective copyright owner
+ * see the NOTICE file and/or the repository at
+ * https://github.com/hyperledger-labs/business-partner-agent
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.hyperledger.bpa.client;
+
+import java.util.Map;
+
+import javax.inject.Singleton;
+
+import io.micronaut.context.annotation.Requires;
+import io.micronaut.health.HealthStatus;
+import io.micronaut.management.endpoint.health.HealthEndpoint;
+import io.micronaut.management.health.indicator.AbstractHealthIndicator;
+import io.micronaut.management.health.indicator.annotation.Liveness;
+
+@Singleton
+@Liveness
+@Requires(beans = HealthEndpoint.class)
+public class BPALivenessCheck extends AbstractHealthIndicator<Map<String, String>> {
+    @Override
+    protected Map<String, String> getHealthInformation() {
+        this.healthStatus = HealthStatus.UP;
+        return null;
+    }
+
+    @Override
+    protected String getName() {
+        return "aca-py-liveness";
```

Comment:
Ok, can you tell me again why we need this? Now it is totally confusing. The BPA already has a liveness check under /health. So now there is a aca-py status below that that always returns true. If something always returns true its a strong indicator you don't need it. Especially if aca-py is down and the BPA is unusable it will still return true. If you really mean the aca-py status we should return the aca-py status and use the one from acaPyClient.statusLive().